mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -559,6 +560,9 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 		METHOD method_26234 isFullCube (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Z
 			ARG 1 world
 			ARG 2 pos
+		METHOD method_27851 testIfInTag (Lnet/minecraft/class_3494;Ljava/util/function/Predicate;)Z
```

Comment:
```suggestion
		METHOD method_27851 isIn (Lnet/minecraft/class_3494;Ljava/util/function/Predicate;)Z
```
`isIn` should suffice

Reply:
Fixed in 3ce10eccb2f0ed761fbf6162a71e12797f95f234